### PR TITLE
(cmake)(cmake.install) Clarify the text around the PATH

### PR DESCRIPTION
**Pull request by pauby**
## Description
I have added text based on #2442 to make it clearer the `cmake.exe` is not on the PATH as it is following the installer behaviour, and providing options.

## Motivation and Context
See #2442

## How Has this Been Tested?
This is README changes and `Write-Warning`s. No other code changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [x] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] I have updated the package description and it is less than 4000 characters.
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey-community/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the Chocolatey Test Environment(https://github.com/chocolatey-community/chocolatey-test-environment/). _Note that we don't support the use of any other environment_.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/cmake.install/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -15,3 +15,7 @@ $packageArgs = @{
 Install-ChocolateyInstallPackage @packageArgs
 
 Get-ChildItem $toolsPath\*.msi | ForEach-Object { Remove-Item $_ -ea 0; if (Test-Path $_) { Set-Content "$_.ignore" '' } }
+
+Write-Warning "Note that this package follow the default behaviour of the 'cmake' installer and does not add cmake.exe to the PATH."
```
Comment (gep13) [inline automatic/cmake.install/tools/chocolateyInstall.ps1:19]:
```suggestion
Write-Warning "Note that this package follows the default behaviour of the 'cmake' installer and does not add cmake.exe to the PATH."
```

File: automatic/cmake/Readme.md [hunk 2]
```diff
@@ -18,4 +19,9 @@ CMake is an open-source, cross-platform family of tools designed to build, test
 - `0` = Install for the current user only
 - `1` = Install for all users (default)
 
-For example: `choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'`
+For example: `choco install cmake --apply-install-arguments-to-dependencies 'ADD_CMAKE_TO_PATH=System'`
```
Comment (gep13) [inline automatic/cmake/Readme.md:22]: Shouldn't this be?

```suggestion
For example: `choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System' --apply-install-arguments-to-dependencies`
```

**Review by gep13 — CHANGES_REQUESTED** [review]

File: automatic/cmake/Readme.md (hunk 2 above)
Comment (pauby) [reply automatic/cmake/Readme.md]: Fixed.

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by gep13** [comment]
@pauby thanks for getting this fixed up!

**Pull request merged**